srcpkgs/qperf/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'qperf'
 pkgname=qperf
 version=0.4.11
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="automake"
+makedepends="rdma-core-devel libibmad-devel"
+depends="rdma-core libibmad"
```

Comment:
These should be detected automatically. Check what xbps-src prints out after

`... running pre-pkg hook: 04-generate-runtime-deps ...`

Reply:
Fixed.  Thanks for the tip.

Reply:
Here is the revelant section of our manual https://github.com/void-linux/void-packages/blob/master/Manual.md#deps_runtime